doc/source/data/getting-started.rst

```diff
@@ -1,229 +1,103 @@
 .. _datasets_getting_started:
 
-===============
 Getting Started
 ===============
 
+A Ray :class:`Dataset <ray.data.Dataset>` is a distributed data collection. It holds
+a list of Ray object references pointing to distributed data **blocks**, and has APIs
+for distributed data loading and processing. Each block holds an ordered collection
+of items in the format of an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__
+or a Python list.
```

Comment:
@ericl Why the removal of the Pandas block call-out?

Reply:
Pandas is an implementation detail, and should not be exposed to the user.

Reply:
It looks Arrow as a data format is actually more internal?
Maybe what you say is Arrow/Python is *canonical format* in Ray Datasets whereas Pandas is something atop?

Reply:
Let's just keep the wording as is for now... We can migrate to pure Arrow and get rid of Python list in the future anyway.